### PR TITLE
Responsiveness

### PR DESCRIPTION
**Pull request by Megha-Joshi**
In this PR, I have added responsiveness to the sports-cave.

### CODE REVIEW

**Review by srishtigautam21 — APPROVED** [review]
The code is clean and understandable
Suggestion:
change the default login and signup credentials 
good to merge with changes

File: src/cave-components/Homepage/homepage.css [hunk 1]
```diff
@@ -31,10 +31,24 @@
     display: flex;
     flex-direction: column;
     align-items: center;
+    gap: 0.5rem;
 }
 
 .hero-image {
-    margin-bottom: 1.25rem;
+    display: flex;
+    /* margin: 1.25rem; */
```
Comment (zoyatanweer) [inline src/cave-components/Homepage/homepage.css:39]: remove commented code

**Review by zoyatanweer — APPROVED** [review]
code looks clean and good. Good to merge

File: src/cave-components/Homepage/homepage.css (hunk 1 above)
Comment (Megha-Joshi) [reply src/cave-components/Homepage/homepage.css:39]: Okay

**Pull request merged**